client.go

```diff
@@ -652,10 +654,8 @@ func (a *Agent) Reuse() *Agent {
 // certificate chain and host name.
 func (a *Agent) InsecureSkipVerify() *Agent {
 	if a.HostClient.TLSConfig == nil {
-		/* #nosec G402 */
-		a.HostClient.TLSConfig = &tls.Config{InsecureSkipVerify: true} // #nosec G402
+		a.HostClient.TLSConfig = &tls.Config{InsecureSkipVerify: true} //nolint:gosec // We explicitly let the user set insecure mode here
 	} else {
-		/* #nosec G402 */
```

Comment:
This is an issue with `gosec` which has been fixed and will soon be released: https://github.com/securego/gosec/issues/908